packages/table/src/react/index.tsx

```diff
@@ -241,11 +246,51 @@ const TableRow = forwardRef<HTMLTableRowElement, TableRowProps>(
 )
 TableRow.displayName = 'Table.Row'
 
+interface TableDrawerProps extends Omit<HTMLPropsFor<'tr'>, 'ref'> {
+  expanded: boolean
+  colSpan: number
+  indentWithCell?: boolean
```

Comment:
Is this what supports the drawer caret alignment (right/left)?

Reply:
yes. if the indentation is on the left there is an option to insert an empty cell to help with the alignment